config/resources/document-domain.lisp

```diff
@@ -103,3 +103,13 @@
   :resource-base (s-url "http://themis.vlaanderen.be/id/stuk/")
   :features `(include-uri)
   :on-path "pieces")
+
+(define-resource minutes (piece)
+  :class (s-prefix "ext:Notulen")
+  :properties `((:value                 :string   ,(s-prefix "prov:value")))
+  :has-one `((meeting                   :via ,(s-prefix "besluitvorming:heeftNotulen")
+                                        :inverse t
+                                        :as "minutes-for-meeting"))
```

Comment:
As discussed, we can add a `has-one` relationship to `piece-part` here, so the content versioning can differ from the `piece` versioning